code/modules/atmospherics/machinery/components/electrolyzer/electrolyzer_reactions.dm

```diff
@@ -60,4 +60,27 @@ GLOBAL_LIST_INIT(electrolyzer_reactions, electrolyzer_reactions_list())
 	air_mixture.assert_gases(/datum/gas/hypernoblium, /datum/gas/antinoblium)
 	var/proportion = min(air_mixture.gases[/datum/gas/hypernoblium][MOLES], (1.5 * (working_power ** 2)))
 	air_mixture.gases[/datum/gas/hypernoblium][MOLES] -= proportion
-	air_mixture.gases[/datum/gas/antinoblium][MOLES] += proportion * 0.5
+	air_mixture.gases[/datum/gas/antinoblium][MOLES] += proportion * 0.05
```

Comment:
was this intended?

Reply:
Ops was thinking of changing this one to lower it a bit (is antinob, should not be this easy, and I was supposed to put it at 0.01) will revert it